src/components/verification-grid-tile/css/style.css

```diff
@@ -75,3 +75,56 @@
   background-color: var(--oe-panel-color);
   border-radius: var(--oe-border-rounding);
 }
+
+.progress-meter {
+  --progress-meter-rounding: var(--oe-border-rounding);
+
+  position: relative;
```

Comment:
not inline-block?

Reply:
We discussed offline. `position: relative` is not needed and `display: inline-block` causes the grid tiles to grow too large (further investigation required).

`display: flex` was chosen because it allows the CSS engine to do segmentation